uberfire-project/uberfire-project-client/src/test/java/org/guvnor/common/services/project/client/repositories/ConflictingRepositoriesPopupTest.java

```diff
@@ -126,44 +111,56 @@ public void testSetContent_Reuse() {
                              metadata,
                              command);
 
-        verify(view,
-               times(1)).clear();
-        verify(view,
-               times(1)).setContent(eq(gav),
-                                    eq(metadata));
-        verify(view,
-               times(1)).addOKButton();
-        verify(view,
-               times(1)).addOverrideButton(any(Command.class));
+        verify(view).clear();
+        verify(view).setContent(eq(gav), eq(metadata));
+        verify(view).addOKButton();
+        verify(view).addOverrideButton();
 
         //Re-use
-        presenter.setContent(gav,
-                             metadata,
-                             command);
-        verify(view,
-               times(2)).clear();
-        verify(view,
-               times(2)).setContent(eq(gav),
+        presenter.setContent(gav, metadata, command);
+        verify(view, times(2)).clear();
+        verify(view, times(2)).setContent(eq(gav),
                                     eq(metadata));
-        verify(view,
-               times(2)).addOKButton();
-        verify(view,
-               times(2)).addOverrideButton(any(Command.class));
+        verify(view, times(2)).addOKButton();
+        verify(view, times(2)).addOverrideButton();
     }
 
     @Test
     public void testShow() {
         presenter.show();
 
-        verify(view,
-               times(1)).show();
+        verify(view).show();
     }
 
     @Test
     public void testHide() {
         presenter.hide();
 
-        verify(view,
-               times(1)).hide();
+        verify(view).hide();
+    }
+
+    @Test
+    public void testActions() {
+        final Command okCommand = mock(Command.class);
+        final Command overrideCommand = mock(Command.class);
+
+        final GAV gav = mock(GAV.class);
+
+        final Set<MavenRepositoryMetadata> metadata = new HashSet<MavenRepositoryMetadata>();
+        final Set<Role> roles = new HashSet<Role>() {{
```

Comment:
nitpick: Since Java 8 you can just do
```Java
final Set<Role> roles = Collections.singleton(new RoleImpl(AppRoles.ADMIN.getName()));
```
to save a few keystrokes (and one extra anonymous inner class)

Reply:
good point thanks!